src/query/query_parser/query_parser.rs

```diff
@@ -847,6 +846,10 @@ fn convert_to_query(fuzzy: &FxHashMap<Field, Fuzzy>, logical_ast: LogicalAst) ->
     }
 }
 
+fn supports_ff_range_queries(field_type: &FieldType) -> bool {
+    matches!(field_type, FieldType::Date(_) | FieldType::IpAddr(_) | FieldType::U64(_) if field_type.is_fast())
```

Comment:
IP or any type that maps to u64 is supported.

You can reuse that method:
https://github.com/quickwit-oss/tantivy/pull/1762/files#diff-7041df7b5853bd1b4eb09be3f79a4fbe0ae29301c7d40e192c6589c95fd2c957R299

Reply:
Thanks for the pointer. Fixed.